src/ValueObject/TwigAutocomplete/DemoLink.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\Website\ValueObject\TwigAutocomplete;
+
+/**
+ * Only helper Twig autocomplete class
+ */
+final class DemoLink
```

Comment:
ValueObjects should be always immutable, is it possible to make this object without public properties?

Reply:
Agreed. It's only for twig autocomplete of array, not real value object.

Reply:
Agreed. It's only for twig autocomplete of array, not real value object.

Here: https://github.com/rectorphp/getrector.org/blob/master/templates/homepage/demo.twig#L20

Any idea how to make it work and keep code clean?